javascript/javascript1/week2/Candy_add/candy.js

```diff
@@ -0,0 +1,53 @@
+let boughtCandyPrices = [];
+let amountToSpend = Math.random() * 100;
+function addCandy(candyType, weight) {
+    
+    let candyPrice = 0;
+
+    if (candyType === "Sweet") {
+        candyPrice = weight * 0.5;
+    }
+    else if (candyType === "Chocolate") {
+        candyPrice = weight * 0.7;
+    }
+    else if (candyType === "Toffee") {
+        candyPrice = weight * 1.1;
+    }
+    else if (candyType === "Chewing-gum") {
+        candyPrice = weight * 0.03;
+    }
+    else{
+        console.log("You cannot buy this candy!");
+    }
+    
+    if(candyPrice >0)
+    {
+        boughtCandyPrices.push(candyPrice);
+    }
+    return candyPrice;
+}
+
+function canBuyMoreCandy() {
+    let totalPrice=0;
+    let canBuyMore = true;
```

Comment:
I don't think we need this variable since we don't do anything with its return value. You can stick to using the [comparison operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators) to evaluate your conditional statements.